addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxCubeBridgeHandler.java

```diff
@@ -520,41 +617,34 @@ private synchronized boolean sendCubeCommand(CubeCommand command) {
      * @param terminator String with ending messagetype e.g. L:
      * @throws IOException
      */
-    private void readliness(String terminator) throws IOException {
-        if (terminator == null) {
-            return;
-        }
-        boolean cont = true;
-        while (cont) {
+    private void readLines(String terminator) throws IOException {
+        while (true) {
```

Comment:
Yes, but I was actually refering to [`Thread.isInterrupted()`](https://docs.oracle.com/javase/7/docs/api/java/lang/Thread.html#isInterrupted()) if some other process interupts your thread. But looking close at the code its just a while input is received.